pkgs/development/interpreters/guile/3.0.nix

```diff
@@ -126,9 +131,6 @@ builder rec {
   #  https://github.com/NixOS/nixpkgs/pull/160051#issuecomment-1046193028
   ++ lib.optional (stdenv.hostPlatform.isDarwin) "--disable-lto";
 
-  # Fix build with gcc15
```

Comment:
you lost this comment line, otherwise lgtm

Reply:
Good catch! Restored comment (and moved the other libgcc_s related comment closer again as well).